### ISSUE TITLE
Show user with release notes once extension release is done

### ISSUE THREAD

**Issue by invincibleJai** [fabric8-analytics/fabric8-analytics-vscode-extension#311]
**Is your feature request related to a problem? Please describe.**
Show user with changelog once extension release is done as in other extension

e.g :  
![Screenshot 2019-05-23 at 12 43 24 PM](https://user-images.githubusercontent.com/5129024/58233101-d07da500-7d58-11e9-96f2-75ec6bb4b6ec.png)

https://github.com/eamodio/vscode-gitlens/blob/7f8680232501851840dec328b97fb2a3acd96bce/src/extension.ts#L164

**Comment by arajkumar** [comment]
@SriKrishnaPaparaju  This is a non functional and nice to have feature, not a must have. Shall we defer it from 0.13?

**Comment by SriKrishnaPaparaju** [comment]
@arajkumar  its important notification that we would like the users to be aware of a new release.
